src/main/java/org/phyloref/jphyloref/helpers/PhylorefHelper.java

```diff
@@ -23,22 +18,40 @@
  */
 public class PhylorefHelper {
     // IRIs used in this package.
+	
+	/** IRI for OWL class Phyloreference */
     public static final IRI IRI_PHYLOREFERENCE = IRI.create("http://phyloinformatics.net/phyloref.owl#Phyloreference");
+    
+    /** IRI for the OWL object property indicating which phylogeny a node belongs to */
     public static final IRI IRI_PHYLOGENY_CONTAINING_NODE = IRI.create("http://vocab.phyloref.org/phyloref/testcase.owl#in_phylogeny");
+    
+    /** IRI for the OWL data property indicating the label of the expected phyloreference */ 
     public static final IRI IRI_NAME_OF_EXPECTED_PHYLOREF = IRI.create("http://vocab.phyloref.org/phyloref/testcase.owl#expected_phyloreference_named");
+
+    /** IRI for the OWL object property indicating which specifiers had not been matched */
     public static final IRI IRI_PHYLOREF_UNMATCHED_SPECIFIER = IRI.create("http://vocab.phyloref.org/phyloref/testcase.owl#has_unmatched_specifier");
+
+    /** IRI for the OWL data property with the verbatim clade definition */  
     public static final IRI IRI_CLADE_DEFINITION = IRI.create("http://vocab.phyloref.org/phyloref/testcase.owl#clade_definition");
     
+    /**
+     * Extract a list of phyloreferences, i.e. instances of IRI_PHYLOREFERENCE that are punned as OWL classes.
+     *  
+     * @param ontology The ontology to extract the phyloreferences from
+     * @param reasoner A reasoner to use in extracting the phyloreferences
+     * @return A Set of OWLNamedIndividuals corresponding to phyloreferences in the specified ontology
+     */
     public static Set<OWLNamedIndividual> getPhyloreferences(OWLOntology ontology, OWLReasoner reasoner) {
         // Get a list of all phyloreferences. First, we need to know what a Phyloreference is.
         Set<OWLEntity> set_phyloref_Phyloreference = ontology.getEntitiesInSignature(IRI_PHYLOREFERENCE);
         if (set_phyloref_Phyloreference.isEmpty()) {
-            throw new RuntimeException("Class 'phyloref:Phyloreference' is not defined in ontology.");
+            throw new RuntimeException("Class " + IRI_PHYLOREFERENCE + " is not defined in ontology.");
         }
```

Comment:
Is this a programmer or internal error, or is it an illegal argument case? If the latter, use the appropriate exception.